src/Stryker.Core/Stryker.Core/Parsers/ITotalNumberOfTestsParser.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Stryker.Core.Parsers
```

Comment:
This interface seems to only be for dependency injection. Consider placing it in the implementation cs file unless it's actually intended to have multiple implementations.

Reply:
Done